components/patient-data/rest/src/test/java/org/phenotips/data/rest/DateTimeAdapterTest.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/
+ */
+package org.phenotips.data.rest;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+/**
+ * Test for the {@link DateTimeAdapter} class
+ */
+public class DateTimeAdapterTest {
+
+    private DateTimeAdapter adapter;
+
+    private DateTime dateTime;
+
+    private String ValidDateString;
+
+
+    @Before
+    public void setUp() throws Exception
+    {
+        this.adapter = new DateTimeAdapter();
+        this.dateTime = new DateTime(2015, 1, 1, 12, 0, DateTimeZone.UTC);
+        this.ValidDateString = "2015-01-01T12:00:00.000Z";
```

Comment:
[It is currently written like this](https://github.com/phenotips/phenotips/blob/master/components/patient-data/rest/src/main/java/org/phenotips/data/rest/DateTimeAdapter.java#L46)